### ISSUE TITLE
CRON monitor support

### ISSUE THREAD

**Issue by requiming** [site24x7/terraform-provider-site24x7#238]
Hi,

Do you plan to add terraform resource support to create this monitor?

Thank you.

**Comment by anita-mithran** [comment]
Yes. we are taking this as a feature request and have started working on this. Will update you in the thread, once we release it in the github.

**Comment by ani3588** [comment]
Hi requiming.. My apologies on missing your comment. We have completed cron monitor support in terraform and have raised a MR for it. We will merge the request soon for your use. 

Thank you for you patience.
